Tests/Resources/app/AppKernel.php

```diff
@@ -10,10 +10,16 @@ class AppKernel extends Kernel
      */
     public function registerBundles()
     {
-        return [
+        $bundles =  [
             new \Symfony\Bundle\FrameworkBundle\FrameworkBundle(),
             new \Http\HttplugBundle\HttplugBundle(),
         ];
+
+        if (false === defined('HHVM_VERSION')) {
+            array_splice($bundles, 1, 0, [new \Puli\SymfonyBundle\PuliBundle()]);
+        }
```

Comment:
> Well, rethinking this: it does not, since all compilers are loaded after
all extensions did.

exactly, one of the principal reasons to use a compiler pass. you can even
in the extension class see which bundles are instantiated, but not their
config